build/media-suite_compile.sh

```diff
@@ -1916,7 +1927,7 @@ if [[ $ffmpeg != "no" ]]; then
             "$LOCALDESTDIR"/bin-video/{sw{scale,resample},postproc}.lib
             )
         _check=()
-        sedflags="prefix|bindir|extra-version|pkg-config-flags"
+        sedflags="prefix|bindir|extra-\w*|pkg-config-flags|disable-debug|shlibdir"
```

Comment:
Why are you removing disable-debug? This is imperative to leave in the compiled binary for proper issues to reported to FFmpeg.
These modifications are unrelated to this PR and should proposed in a separate PR or issue.

Reply:
You can leave it as you wish.